tests/vmi_networking_test.go

```diff
@@ -55,7 +55,6 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	var virtClient kubecli.KubevirtClient
 
 	var inboundVMI *v1.VirtualMachineInstance
```

Comment:
Also some test under https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_networking_test.go#L676 can be deleted with this PR I think.

Reply:
The other two cases are for external connectivity, which is not covered in this test. It should follow in other PRs. We should reconsider whether we want to ping to the outside network at all.